code/API_definitions/connected-network-type.yaml

```diff
@@ -0,0 +1,405 @@
+openapi: 3.0.3
+info:
+  title: Connected Network Type
+  description: |
+    This API provides the customer with the ability to query to which Mobile Communication Technology it is connected to.
+
+    # Introduction
+
+    ## Connected Network Type
+
+    API consumer is able to inquire device's connected network type.
+
+    # Relevant terms and definitions
+
+    * **Device**: A device refers to any physical entity that can connect to a network and participate in network communication.
+      At least one identifier for the device (user equipment) out of four options: IPv4 address, IPv6 address, Phone number, or Network Access Identifier assigned by the mobile network operator for the device.
+
+    * **Network Type** : Network Type is intended to provide insight to connected network's capabilities from standards perspective.  Actual network capabilities may differ based on implementation and MUST be checked with the connected network provider.
+      - `2G`, if device is connected to the 2G network technology
```

Comment:
@eric-murray  Would it be better to include '_**Displayed connected technology indicator on device may differ from the types returned by this API**_' in the Network Type definition, and don't list out any specific values.  Such a list may be interpreted as all possible values.

Reply:
I think the values I listed should cover most devices, but I agree it may well not be comprehensive. I often used to get asked "What does "E" or "H+" on my phone mean?", but not so much these days.

How about something like "alternative indicators such as "G" or "E" may be displayed on the device" ? So listing the common options, but saying these are just examples.

Reply:
Sounds good to me. Thx!